packages/workflow-transformer-async/flows/ReactExample.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env node */
+import React from 'react';
+import render, { Workspace, requireComponent, createComponentRecursive } from 'workflow-react';
+import { Terminal } from 'workflow-apps-defaults';
+
+const TerminalAsync = createComponentRecursive({
```

Comment:
I have kind of viewed these examples as tests, rather than actual examples. However, I think the use case is valid, but not as a pure wrapper more like there is need to do async operation to define components. I guess whether it is worth supporting this depends on how often apps will be defined inline vs in its own module when modding stuff locally in workflow home.

Reply:
I could solve the same thing by adding `AsyncTerminal.js` in the flows folder and use requireComponent to import it.

Reply:
Yeah, I really want the duality of executable examples as test. :) So they should serve both purposes. So we should strive for best practices :+1: 

I'd wanna make a case for using the [`.workflow/apps/user`](https://github.com/havardh/workflow/tree/master/packages/create-workflow-home/files/static#appsuserjs) in case you want to add `AsyncTerminal.js`, just adding the reference.

Reply:
How can the example in the repo use `.workflow` files?

Reply:
We need to hash this out but I'm am considering doing:

`npx create-workflow-home packages/workflow-home-example`

and making all examples use `--config packages/workflow-home-example/config.js` 

That would make `packages/workflow-home-example/apps/user` available with `import App from "../../workflow-home-example/apps/user`, hm... I was thinking this could be `from "apps/user"`  or `from "./apps/user` but all of that is to magical. I have considered something like `import x from "WORKFLOW_HOME/...` but I decided against it. Maybe the answer is that you can/should not access `.workflow` from the examples in the repo. Or we could do it only from the examples in `workflow-home-example` :thinking: